lib/spotlight/dor/indexer.rb

```diff
@@ -50,6 +52,12 @@ def add_box(sdb, solr_doc)
         solr_doc['box_ssi'] = sdb.smods_rec.box
       end
 
+      # pulls from //subject/geographic
+      def add_geonames(sdb, solr_doc)
+        ids = extract_geonames_ids(sdb)
+        solr_doc['geographic_srpt'] = ids.map { |id| get_geonames_api_envelope(id) }.reject(&:nil?)
```

Comment:
So these are ok:
- `reject(&:invalid?)`
- `reject(&:blank?)`
- `reject(&:negative?)`
- `reject(&:new?)`

But `reject(&:nil?)` is not native-enough ruby?  You can understand it perfectly well, and you can tolerate `reject` in all other cases... except this one.